gnosis/safe-react
# feat: improve analytics

## What it solves
Resolves #3408

## How this PR fixes it
- Google Tag Manager implementation
- Update cookie banner to load GTM (#3417)
- Create tracking HoC (#3421)
- Anonymous tracking (#3426)
- Removal of GA (https://github.com/gnosis/safe-react/pull/3613)
- Top-level tracking (https://github.com/gnosis/safe-react/pull/3659)

## Events
- Safe creation/load (#3553)
- Safe 'overview' (#3606)
- All requested events (https://github.com/gnosis/safe-react/pull/3675)

## How to test
- Open the Safe and observe the the GTM script only injects when the analytics cookie checkbox is accepted
- Open the Safe in GTM preview
- Observe tracking of Safe creation/loading and overview according to the first two sections [here](https://github.com/gnosis/safe-react/issues/3408) in the console info
